src/main/scala/scalajsGames/games/Snake.scala

```diff
@@ -83,7 +86,9 @@ case class Snake(bounds: Point, resetGame: () => Unit) extends Game {
           resetGame()
         case x =>
           x match {
-            case Apple(_, s) => length += s
+            case Apple(_, s) =>
+              score += s / 2
```

Comment:
I wonder how the score grows and why do we choose that?

Reply:
I saw that there are two cases of apples (red = 2 pts, bonus = 5 pts), so instead of adding 2 or 5 to the score I think that it's better way to add 1 or 2 (dividing 2 and 5 by 2) because it's easier to the user to know what are the value of the apples and how many apples does the user need to get a higher score

Reply:
Sounds reasonable, it is worth commenting this into the code